webrender/src/resource_cache.rs

```diff
@@ -956,13 +958,19 @@ impl ResourceCache {
             .image_templates
             .images
             .retain(|key, _| key.0 != namespace);
+        self.cached_images
+            .clear_keys(|request| request.key.0 == namespace);
 
+        self.resources.font_instances
+            .write()
+            .unwrap()
+            .retain(|key, _| key.0 == namespace);
+        for &key in self.resources.font_templates.keys().filter(|key| key.0 == namespace) {
```

Comment:
This reads incorrect to me: first we retain "namespace"  only keys (already wrong - we should retain everything *but* "namespace" keys), and then we go through them, filtering on the same criteria again.

Reply:
Accidental off-by-one-character bug. This should be fixed in PR #2520.